modules/reitit-frontend/src/reitit/frontend.cljs

```diff
@@ -40,7 +51,14 @@
 
 (defn router
   "Create a `reitit.core.router` from raw route data and optionally an options map.
-  Enables request coercion. See [[reitit.core/router]] for details on options."
+  Enables request coercion. See [[reitit.core/router]] for details on options.
+
+  Additional options:
+
+  | key          | description |
+  | -------------|-------------|
+  | :trailing-slash-handling | TODO |
```

Comment:
could be just `:trailing-slash` (or `:trailing-slash-method` (the ring-version has `:method`)).

Reply:
TODO -> https://github.com/metosin/reitit/blob/master/modules/reitit-ring/src/reitit/ring.cljc#L117-L119